lib/bubbles.js

```diff
@@ -5,10 +5,10 @@ function Bubble(x, y, direction, canvas){
   this.height = 5;
   this.width = 5;
   if (direction === "right") {
```

Comment:
Maybe we can extract this statement into another method so that the constructor is cleaner?